geode-core/src/distributedTest/java/org/apache/geode/internal/cache/RegionConcurrentOperationDUnitTest.java

```diff
@@ -61,9 +63,14 @@ private static DUnitBlackboard getBlackboard() {
   @Rule
   public ExecutorServiceRule executorServiceRule = new ExecutorServiceRule();
 
+  @Rule
+  public DistributedBlackboard blackboard = new DistributedBlackboard();
+
   @After
   public void tearDown() {
-    blackboard.initBlackboard();
+    if (blackboard != null) {
```

Comment:
You can delete this `tearDown`. The `DistributedBlackboard` Rule handles this for you. (If it doesn't, that's a bug that I need to know about.)